### DISCUSSION TITLE
Laser starts cutting while traveling to image

### DISCUSSION THREAD

**Discussion by Radfordfam** [arkypita/LaserGRBL#1260] [Q&A]
My laser starts cutting while traveling to image location. I have read over other discussions in this issue but i do not have $32 in my grbl configuration it only goes to $22 so i dont know how to change that to 1.
![16103320019682532860944653525634](https://user-images.githubusercontent.com/77255222/104142682-cff4b080-5379-11eb-813c-d6b46518ed09.jpg)

**Comment by StuartB4** [comment]
You may need to enable laser mode.

When LaserGRBL is connected go to the Grbl menu and click Grbl configuration.
Look for $32 in the list that pops up and if it's set to 0 change it to 1, and set $30 to 255.
Click Write and then close.

**Reply by Radfordfam** [reply]
Ok we got that issue fixed. Now since updating to version 1.1 the laser will come on, but will not burn. Our settings are
$0=10
$1=25
$2=0
$3=6
$4=0
$5=0
$6=0
$10=3
$11=0.010
$12=0.002
$13=0
$20=0
$21=0
$22=0
$23=0
$25=500
$26=250
$27=1.0
$30=255
$31=0
$32=1
$100=250
$101=250
$102=350
$110=400
$111=400
$112=400
$120=10
$121=10
$122=10
$130=200
$131=200
$132=200

**Comment by StuartB4** [comment]
Your settings look fine. Do you have S-MAX in Target image settings set high enough. It should be 255 for maximum power. Or anywhere between 0 and 255. Also is the laser focussed properly?

![image](https://user-images.githubusercontent.com/16980026/104181910-7dfa6c00-5407-11eb-953d-27b276ff0034.png)

**Reply by Radfordfam** [reply]
Yes s-max is at 255. The laser is on but it wont focus

**Comment by StuartB4** [comment]
Does it have the adjusting ring at the bottom of the laser or is it a fixed focus laser?

**Comment by Radfordfam** [comment]
It has the adjusting ring

**Comment by StuartB4** [comment]
When you say it does'nt focus do you mean you can't make the laser dot smaller, or it won't switch the laser on to focus?

**Comment by Radfordfam** [comment]
We can focus it to a small dot but defore we updated when we hit focus it would go bright, then dim to where it needed to be. It will no longer do that since the update. The laser is on but its really dim, its not getting enough power or something.

**Reply by arkypita** [reply]
Focus power is controlled by the gcode you have in custom button.
What is your code? Right click on focus button, edit button.

**Comment by regpye** [comment]
I am having a similar problem.
It didn't do this before, just started doing it today.
I have checked all my settings and configurations and all seem to be OK
The laser turns on before it reaches the cutting area and continues cutting all the time when it should turn off between various sections of the job.
At the end of the job it will turn off OK

$0=10 (Step pulse time)
$1=25 (Step idle delay)
$2=0 (Step pulse invert)
$3=0 (Step direction invert)
$4=0 (Invert step enable pin)
$5=0 (Invert limit pins)
$6=0 (Invert probe pin)
$10=1 (Status report options)
$11=60.000 (Junction deviation)
$12=0.002 (Arc tolerance)
$13=0 (Report in inches)
$20=0 (Soft limits enable)
$21=0 (Hard limits enable)
$22=1 (Homing cycle enable)
$23=3 (Homing direction invert)
$24=1200.000 (Homing locate feed rate)
$25=1000.000 (Homing search seek rate)
$26=144 (Homing switch debounce delay)
$27=3.000 (Homing switch pull-off distance)
$30=1000 (Maximum spindle speed)
$31=0 (Minimum spindle speed)
$32=1 (Laser-mode enable)
$100=40.000 (X-axis travel resolution)
$101=40.000 (Y-axis travel resolution)
$102=1600.000 (Z-axis travel resolution)
$110=5000.000 (X-axis maximum rate)
$111=5000.000 (Y-axis maximum rate)
$112=150.000 (Z-axis maximum rate)
$120=1000.000 (X-axis acceleration)
$121=1000.000 (Y-axis acceleration)
$122=100.000 (Z-axis acceleration)
$130=215.000 (X-axis maximum travel)
$131=230.000 (Y-axis maximum travel)
$132=200.000 (Z-axis maximum travel)

The beginning of the file looks like this:
M3 S0
S0 
G0X1.5Y41.7
S700 
G1X1.7Y41.3F1000
X2.3Y40.7
Y40.6
X2.6Y40.3
X3.6Y39.3
 and the end looks like this:
Y2.2
X24.6Y2.6
Y3
X24.5Y3.1
X24.3Y3.9
S0 
M5 S0

**Reply by StuartB4** [reply]
Does it do it when you use M4 instead of M3 for Laser On. 

What machine is it. I know that Ortur have had problems with their motherboards. 
Some people have problems with the laser leaving burn lines between objects or letters
even when $32 (laser mode) is enabled.  Ortur have a new board out now.
